packages/client/src/proxy.js

```diff
@@ -169,6 +169,7 @@ export class ProxyHttpsAgent extends https.Agent {
     let handleError = err => {
       socket.destroy(err);
       logger('client:proxy').error(`Proxying request failed: ${err}`);
+      logger('client:proxy').error('When using a proxy, please configure the following environment variables: HTTP_PROXY, HTTPS_PROXY, and NO_PROXY.');
```

Comment:
- Proxy auth creds are correct - 401/403 
- Proxy is correctly set and reachable 
- The domains percy.io/github/percy-enterprise.browserstack/storage.google.com are whitelisted on proxy

Reply:
Resolved.
- Added warning message for the same in `proxy.js`